### PR TITLE
[DA] added timer Start/Cancel/Status and AddItem

### PR DESCRIPTION
**Pull request by nic0dk**
[DA] _common.yaml 
- added wildcard for shopping_list_item
- timer seconds/minutes/hours/half/quarter
- expansion rules timer pause/add/decrease/set/cancel/duration/start

[DA] homeassistant_HassStartTimer.yaml
[DA] todo_HassListAddItem.yaml
[DA] homeassistant_HassCancelTimer.yaml
[DA] homeassistant_HassTimerStatus.yaml
[DA] homeassistant_HassNevermind.yaml

### CODE REVIEW

**Review by MTrab — APPROVED** [review]
Looks good to me

**Review by MTrab — CHANGES_REQUESTED** [review]
Tests are missing for these files

File: tests/da/homeassistant_HassCancelTimer.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - "stop nedtællingen"
+      - "afbryd nedtælling"
+      - "afbryd min nedtælling"
+      - "fjern nedtælling"
+      - "fjern min nedtælling"
+      - "slet nedtælling"
+      - "slet min nedtælling"
+    intent:
+      name: HassCancelTimer
+    response: Nedtælling afbrydt
```
Comment (MTrab) [inline tests/da/homeassistant_HassCancelTimer.yaml:14]: Typo, I guess

File: tests/da/homeassistant_HassCancelTimer.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  - sentences:
+      - "stop 5 minutter nedtælling"
+      - "stop 5 minutter min nedtælling"
+      - "stop nedtællingen på 5 minutter"
+      - "afbryd 5 minutter nædtelling"
+    intent:
+      name: HassCancelTimer
+      slots:
+        start_minutes: 5
+    response: Nedtælling afbrydt
```
Comment (MTrab) [inline tests/da/homeassistant_HassCancelTimer.yaml:24]: Another

File: tests/da/homeassistant_HassCancelTimer.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      - "stop pizza nedtælling"
+      - "afbryd min pizza nedtælling"
+      - "fjern min nedtælling for pizza"
+    intent:
+      name: HassCancelTimer
+      slots:
+        name:
+          - "pizza "
+          - "pizza"
+    response: Nedtælling afbrydt
```
Comment (MTrab) [inline tests/da/homeassistant_HassCancelTimer.yaml:36]: Again

File: tests/da/homeassistant_HassCancelTimer.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+  - sentences:
+      - "fjern nedtælling på køkken"
+      - "slet nedtælling på køkkenet"
+      - "afbryd nedtællingen på køkkenet"
+    intent:
+      name: HassCancelTimer
+      slots:
+        area: Køkken
+    response: Nedtælling afbrydt
```
Comment (MTrab) [inline tests/da/homeassistant_HassCancelTimer.yaml:45]: One more

**Review by MTrab — CHANGES_REQUESTED** [review]
Typos?

File: tests/da/todo_HassListAddItem.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+language: da
+tests:
+  - sentences:
+      - læg æbler til dagligvarer
+      - lægge æbler til i dagligvarer
+      - tilføje æbler til dagligvarer
+      - tilføj æbler til i dagligvarer
```
Comment (MTrab) [inline tests/da/todo_HassListAddItem.yaml:7]: Shouldn't that be `tilføj æbler til dagligvarer` ?

**Comment by nic0dk** [comment]
@MTrab hope we nailed all the typo's, else let me know

**Review by MTrab — APPROVED** [review]

**Pull request merged**